backend/lib/testing/mock-gateways/mock-oauth2-gateway.ts

```diff
@@ -20,13 +21,13 @@ export async function mockAuthentication(context: ApplicationContext): Promise<s
   const validMockRole = mockUsers.find((role) => role.sub === requestedSubject.sub);
 
   const ONE_DAY = 60 * 60 * 24;
-  const SECONDS_SINCE_EPOCH = Math.floor(Date.now() / 1000);
+  const NOW = nowInSeconds();
 
   const claims: CamsJwtClaims = {
     aud: 'api://default',
     sub: validMockRole.sub,
     iss: context.request.url,
-    exp: SECONDS_SINCE_EPOCH + ONE_DAY,
+    exp: NOW + ONE_DAY,
```

Comment:
What do you think about adding an environment variable so that we can manipulate this in the deployed dev environments without code changes? Something like:
```suggestion
    exp: NOW + (process.env.MOCK_SESSION_EXPIRE_LENGTH ?? ONE_DAY),
```